tests/helpers/linux/test_slab.py

```diff
@@ -51,3 +55,9 @@ def test_find_slab_cache(self):
         for name in slab_cache_names:
             slab = find_slab_cache(self.prog, name)
             self.assertEqual(name, slab.name.string_())
+
+    def test_get_allocated_slab_objects(self):
+        type_name = "struct dentry"
+        slab = find_slab_cache(self.prog, "dentry")
+        type_obj = self.prog.type(type_name)
+        self.assertTrue(len(get_allocated_slab_objects(slab, type_obj)) > 0)
```

Comment:
Also as we discussed, we can make this test a bit stronger by checking that some valid dentry is in the list. For example, we can use the dentry for the root directory, `prog["init_fs"].root.dentry`. Something like

```suggestion
        self.assertIn(prog["init_fs"].root.dentry, slab_cache_for_each_allocated_object(slab, "struct dentry"))
```